BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -275,10 +276,11 @@ class GalleryViewController: UIViewController {
     }
     
     override func viewDidAppear(_ animated: Bool) {
-        if !currentUserManager.signedIn(context: viewContext) {
+        guard currentUserManager.signedIn(context: viewContext) else {
             let signInVC = SignInViewController(currentUserManager: currentUserManager)
             signInVC.modalPresentationStyle = .fullScreen
             self.present(signInVC, animated: true, completion: nil)
+            return
```

Comment:
Let's remove this part of the change - It's not relevant to the PR, and it's a kinda surprising use of guard.